platform/src/main/java/org/hillview/sketches/ExactCountSketch.java

```diff
@@ -0,0 +1,80 @@
+package org.hillview.sketches;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenCustomHashMap;
+import it.unimi.dsi.fastutil.objects.Object2IntOpenHashMap;
+import org.hillview.dataset.api.ISketch;
+import org.hillview.table.api.IRowIterator;
+import org.hillview.table.api.ITable;
+import org.hillview.table.rows.RowSnapshot;
+import org.hillview.table.rows.VirtualRowHashStrategy;
+import org.hillview.table.rows.VirtualRowSnapshot;
+import org.hillview.utils.MutableInteger;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+
+public class ExactCountSketch implements ISketch<ITable, FreqKList> {
```

Comment:
Perhaps you should use a different name which makes it clear that this is tied to MinCount.
Also, so comments would be nice.